lnrpc/signrpc/signer.proto

```diff
@@ -0,0 +1,118 @@
+syntax = "proto3";
+
+package signrpc;
+
+message KeyLocator {
+    /// The family of key being identified.
+    int32 key_family = 1;
+
+    /// The precise index of the key being identified.
+    int32 key_index = 2;
+}
+
+message KeyDescriptor {
+     oneof key {
+         /**
+         The raw bytes of the key being identified. Either this or the KeyLocator
+         must be specified.
+         */
+         bytes raw_key_bytes = 1;
```

Comment:
What is the use case for passing in `raw_key_bytes`?

Reply:
When you know the pubkey, but not the key locator information.